extensions/interactions/MathEquationInput/protractor.js

```diff
@@ -19,8 +19,17 @@
 
 var objects = require(process.cwd() + '/extensions/objects/protractor.js');
 
-var customizeInteraction = function() {
-  // There are no customizations.
+var customizeInteraction = async function(elem, customLetters) {
+  expect(await elem.element(
+    by.css('.protractor-test-custom-letters-div')).isPresent()).toBe(true);
+  for (let letter of customLetters) {
+    await elem.element(by.buttonText('abc')).click();
+    let letterIsPresent = await elem.element(by.buttonText(letter)).isPresent();
+    if (!letterIsPresent) {
+      await elem.element(by.buttonText('αβγ')).click();
+    }
+    await elem.element(by.buttonText(letter)).click();
+  }
```

Comment:
Same comments as I left for AlgebraicExpressionInput:

* Use waitFor statements instead of `.isPresent()` to allow for elements needing to load
* Check elements have loaded before clicking them

Reply:
Done.